web/src/containers/StateDashboard.js

```diff
@@ -3,8 +3,6 @@ import PropType from 'prop-types';
 import React, { Fragment } from 'react';
 import { connect } from 'react-redux';
 
-import Footer from '../components/Footer';
-import Header from '../components/Header';
```

Comment:
I originally merged master into this, which made it super messy. It also means I deleted the header, because it's in the wrapper in master. Oops?